contributed_definitions/nyaml/NXapplied_pressure.yaml

```diff
@@ -0,0 +1,22 @@
+category: base
+doc: |
+  Pressure applied to sample.
+symbols:
+  n_shear: |
+    number of values in applied pressure
+type: group
+NXapplied_pressure(NXexternal_stimulus):
```

Comment:
I am convinced having a "depends_on" field for the quantity is useful to specify in which reference frame the quantity is defined. The main need for having something like this is that in continuum mechanics you often tend to distinguish different reference configurations, i.e. frames of references (the deformed the undeformed) coordinate system and being specific in which coordinate system the quantity is defined is vital in such applications. With NeXus and NXcoordinate_system(_set) plus NXtransformations we have all this at our fingertips which is much more powerful than plain ad hoc generated flat lists of key-value pairs like e.g. also used in NOMAD metainfo right now